tools/main.py

```diff
@@ -62,8 +62,6 @@ def main():
     sys.stdout = Logger(osp.join(cfg.data.save_dir, log_name))
 
     print('Show configuration\n{}\n'.format(cfg))
-    print('Collecting env info ...')
-    print('** System info **\n{}\n'.format(collect_env_info()))
```

Comment:
Why did you remove this?

Reply:
In my case, these two lines of code work slowly. I need to wait up to a minute to collect env info. So, since it is not a very necessary thing, I propose to remove it. But there is something wrong with my environment, and on the cluster, it is working pretty fast. I could return it back